### PR TITLE
Prepare release v0.9.0

### PR DESCRIPTION
**Pull request by Odraxs**
This pull request includes updated execution and continuation command documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,12 +97,43 @@ Kadena.Pact.ExecCommand.new()
...
+- [Signers](#signerslist)
+- [Step](#step)
+- [Proof](#proof) (optional)
+- [Rollback](#rollback)
+- [PactId](#pactid)
+
+The following example shows how to create a continuation command:
+
+```elixir
+     Kadena.Pact.ContCommand.new()
```
Comment (juanhurtado10) [inline README.md]: @Odraxs please fix the indentation

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -97,12 +97,43 @@ Kadena.Pact.ExecCommand.new()
...
   |> Kadena.Pact.ExecCommand.set_nonce(nonce)
   |> Kadena.Pact.ExecCommand.set_data(env_data)
-  |> Kadena.Pact.ExecCommand.set_metadata(keypair)
+  |> Kadena.Pact.ExecCommand.set_metadata(meta_data)
   |> Kadena.Pact.ExecCommand.add_keypair(keypair)
   |> Kadena.Pact.ExecCommand.add_signers(signers_list)
   |> Kadena.Pact.ExecCommand.build()
 ```
 
+### Continuation Command
```
Comment (juanhurtado10) [inline README.md:131]: @Odraxs The documents are incomplete, please complete them with examples of each use case. As a user it is not clear to me how to build a command

File: README.md [hunk 3]
```diff
@@ -87,20 +87,107 @@ To create an execution command is needed:
...
+# with a keyword list
+  args = [
+    network_id: network_id,
+    code: code,
+    data: env_data,
+    nonce: nonce,
+    meta_data: meta_data,
+    keypairs: keypairs, #KeyPair list
+    signers: signers #SignersList struct
+  ]
```
Comment (miguelnietoa) [inline README.md]: @Odraxs @EdwinGuayacan please fix the indentation

File: README.md [hunk 4]
```diff
@@ -113,11 +200,30 @@ There are three options allowed to set a NetworkID:
...
+A String value that represents the Pact code to execute in the `Execution Command`.
 
 #### Nonce
 
-String value to ensure unique hash. You can use current timestamp.
+A String value to ensure unique hash. You can use current timestamp.
+
+#### Step
+
+The step is an Integer of the mutli-step transaction.
```
Comment (miguelnietoa) [inline README.md]: @Odraxs @EdwinGuayacan there is a typo here "mutli-step"

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.9.0 (02.12.2022)
```
Comment (miguelnietoa) [inline CHANGELOG.md]: @Odraxs @EdwinGuayacan please change the date

**Review by miguelnietoa — CHANGES_REQUESTED** [review]

**Review by juanhurtado10 — APPROVED** [review]

**Pull request merged**